application/src/main/java/org/togetherjava/tjbot/logging/discord/DiscordLogForwarder.java

```diff
@@ -193,6 +199,14 @@ private static String describeLogEvent(LogEvent event) {
             return logMessage + "\n" + exceptionWriter.toString().replace("\t", "> ");
         }
 
+        private static @Nullable String linkToSource(String source, String sourceCodeBaseUrl) {
+            if (!source.startsWith(BASE_PACKAGE)) {
+                return null;
+            }
+
+            return sourceCodeBaseUrl + source.replace('.', '/') + ".java";
```

Comment:
make sure that the sourceCodeBaseUrl ends with `/`, so that it is ready for a concat.

sth like `if (!sourceCodeBaseUrl.endsWith("/")) sourceCodeBaseUrl += "/"`